spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/SpatialConnect.java

```diff
@@ -228,6 +228,12 @@ public void connectBackend(final SCRemoteConfig remoteConfig) {
         }
     }
 
+    public void connectAuth(ISCAuth authMethod) {
+        authService = new SCAuthService(context, authMethod);
+        addService(authService);
+        startService((SCAuthService.serviceId()));
```

Comment:
Looks like you have some extra parentheses here?  Must be all the LISP you've been reading 😉